src/main/java/io/buildpacks/example/sample/GreetingController.java

```diff
@@ -7,8 +7,14 @@
 @RestController
 public class GreetingController {
 
-    @GetMapping("/greet")
```

Comment:
4. Testing:
   - Make sure to write unit tests to cover the functionality of both the `greeting` and `throwError` methods. Unit tests are essential for ensuring code quality and reliability.